### DISCUSSION TITLE
API Endpoint Down?

### DISCUSSION THREAD

**Discussion by noyez** [modoboa/modoboa#2378] [Q&A]
Hello,

Am i the only one seeing this error, perhaps i have something misconfigured? 

I am getting Cron errors when trying to communicate with the api endpoint https://api.modoboa.org/1/

cmd: `$PYTHON $INSTANCE/manage.py communicate_with_public_api`
output: `CommandError: Failed to retrieve versions from the API.`

```
mba :: ~  » curl https://api.modoboa.org/1/
<h1>Not Found</h1><p>The requested resource was not found on this server.</p>
```
I've seen this before in this issue report https://github.com/modoboa/modoboa/issues/2087.

**Comment by tonioo** [comment]
It can happen sometimes... Right now the API is hosted on a dedicated server but I guess it would be safer to move it on some cloud based environment but I don't have any budget for that.
